agent/rpc/operator/service_test.go

```diff
@@ -0,0 +1,278 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/consul/acl"
+	"github.com/hashicorp/consul/acl/resolver"
+	"github.com/hashicorp/consul/agent/structs"
+	"github.com/hashicorp/consul/proto/pboperator"
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/mock"
+	"google.golang.org/grpc"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type MockBackend struct {
+	mock.Mock
+	authorizer acl.Authorizer
+}
+
+type mockAuthorizer struct {
```

Comment:
What would you think about moving this to an acl/acltest package and allowing the MockAuthorizer to be reused from many places?

Also the acl.Authorizer interface has extra methods in enterprise so this one wont actually meet that interface when this gets merged.

Reply:
I was thinking of reusing the same mock but did not find a good place to move it to be shareable. I think acl/acltest make sense. I will do so.